### ISSUE TITLE
Extra page breaks

### ISSUE THREAD

**Issue by spulakk** [mpdf/mpdf#1544]
### I found this bug
The code below used to generate one page but in the newest version generates `data - page break - blank page - page break - background`. When I wrap the background in a div, it changes to `blank page - page break - data + background`.

### This is mPDF and PHP version and environment (server/fpm/cli etc) I am using
PHP - 8.0.8
MPDF - 8.0.14 (works fine in 8.0.13 and earlier)

### This is the PHP code snippet I use

```
$mpdf = new \Mpdf\Mpdf([
	'mode' => 'utf-8',
	'format' => 'A4',
	'margin_left' => 0,
	'margin_right' => 0,
	'margin_top' => 0,
	'margin_bottom' => 0,
	'orientation' => 'P'
]);

$mpdf->WriteHTML($content);
$mpdf->Output($pdfName, 'I');
```

### This is the HTML/CSS code snippet I use

```
<!DOCTYPE HTML>
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8">
	<title></title>
	<style>
		body {
			font-family: Arial;
		}
		.text {
			font-size: 16px;
			position: absolute;
			z-index:2;
		}
	</style>
</head>
<body>
	<img src="\www\images\background.png">

	<div class="text" style="top:251px;left:65px;">
		123456789
	</div>

	[other data...]
</body>
```
![1_blur](https://user-images.githubusercontent.com/34043062/140058883-30bd396b-bac5-450a-a1f6-6596381c27e5.PNG)

![2_blur](https://user-images.githubusercontent.com/34043062/140058887-edc4ac70-eced-40df-997d-1e2f5c9ddca9.PNG)

**Comment by bomshteyn** [comment]
Just experienced the same issue, and can confirm that downgrading to mpdf 8.0.13 solved it for me

**Comment by PaulodoPorto** [comment]
@spulakk @bomshteyn. 

After two weeks suffering like a dog on a chain to make this to work, I had no success, and just today  11.09.2021 I found this post. I thoght that, this Bug could be something wrong with my code.
As @bomshteyn mentioned, I did the downgrade to v8.0.13 and now to PDF is working again.
It was a really stressfull and terrible time for me. Thank you guys.

**Comment by martingebhardt** [comment]
I have the same issue and downgrading to v8.0.13 helps (bash: composer require mpdf/mpdf:8.0.13).

**Comment by finwe** [comment]
Released 8.0.15 fixing this.
